config/repair.py

```diff
@@ -4,8 +4,7 @@
 # This CR tuning comes from looking at simulated r1 images w/o scattering.
 #
 config.interp.modelPsf.defaultFwhm = 1.75
-config.cosmicray.cond3_fac = 4
-config.cosmicray.cond3_fac2 = 1
+config.cosmicray.cond3_fac = 30
```

Comment:
This looks like a big change in the CR tuning. Is this correct?

Reply:
Yes.